queries.py

```diff
@@ -534,8 +689,10 @@ def construct_unsent_inzendingen_query(max_sending_attempts):
             GRAPH ?h {{
                 OPTIONAL {{ ?decisionType skos:prefLabel ?decisionTypeLabel }} .
             }}
+
+            {2}
         }}
-        """.format(max_sending_attempts, separator.join(allowedDecisionTypesList))
+        """.format(max_sending_attempts, separator.join(allowedDecisionTypesList), verify_inzendingen_flowthrough_rules())
```

Comment:
Sorry this is too complicated; my brain can't parse. 
What's the matter of keeping the query and as we said, for every submission that needs to be sent, check if it matches specific business rules to see if it needs to be excluded?

So basically, right after  https://github.com/lblod/berichtencentrum-sync-with-kalliope-service/blob/master/task_process_inzendingen_voor_toezicht.py#L30
you process the list of inzendingen check if it matches the exclusion criteria, and if so filter them out of the list